### DISCUSSION TITLE
hardcode compensation multipliers

### DISCUSSION THREAD

**Discussion by Draeieg** [ubiquity/ubiquibot#300] [General]
for the implementation of #221 we're currently using a simple number based system, while a simple easy fix for ourselves this could lead to an attack vector once we roll this feature out for partners 

a very unlikely vector that is further hindered by allowance an balance on the permit wallet to the point we could call it a non issue, but if you make a system with the assumption no one using it is a retard someone will prove you wrong break it and then maybe blame you 

for this we can change the use of /setBountyMultiplier from any number to
fulltime=0
halftime=0.5
hunter=1

or alternatively hardcap the /setBountyMultiplier to 1, if /setBountyMultiplier is set to something>1 output an error

**Comment by 0x4007** [comment]
It might be best to output a warning for any value above 1. I don't want to force limit behavior because different organizations have different payment structures. Perhaps they want core contributors to get more than public bounty hunters, for example. In this case you set the bounty pricing to 100 USD but if a core contributor completed it they could get 200 USD.
